### ISSUE TITLE
Orders failing to complete

### ISSUE THREAD

**Issue by NiftyGaloot** [openfoodfoundation/openfoodnetwork#10924]
## Description
Customers are attempting to place orders with shops or hubs and the orders are failing to complete leaving unsatisfied customers and hubs that are not getting orders. Shops and Hubs want a smooth experience for customers and don't want to lose sales when that doesn't happen.

## Expected Behavior
A customer should be able to place an order, go to the cart and check out. The customer should receive an order confirmation and the shop or hub should receive the order. The order should appear as complete when a shop or hub checks their orders page.

## Actual Behaviour
Some customers are placing an order, go to the cart. Attempt to checkout and being redirected without getting confirmation of their order. The shop or hub is not getting notice of the order either. The order is not appearing as complete on the shop or hubs order page and can only be seen by unchecking the "Only Show Complete Orders" box. The order 'State' is shown as "Confirmation"

## Steps to Reproduce
So far have been unable to reproduce the failure to complete an order when testing as a customer.
Have been able to find a number of examples happening.

## Animated Gif/Screenshot
![image](https://github.com/openfoodfoundation/openfoodnetwork/assets/132571876/2c83b97d-064f-4285-b9dd-c9b10ec2083a)
![image](https://github.com/openfoodfoundation/openfoodnetwork/assets/132571876/c6ad2e80-9b12-43da-bde6-eefde37f9686)
![image](https://github.com/openfoodfoundation/openfoodnetwork/assets/132571876/e9255bf6-6b4e-40c6-af3c-a6b42b6f222a)

## Workaround

- Click on the order number of the order that is not complete and showing 'Confirmation'
- Return to the Orders page

## Your Environment
All browsers. USA Instance

**Comment by sigmundpetersen** [comment]
I wonder if this is the same as #10719 ?

**Comment by sigmundpetersen** [comment]
@filipefurtad0 do you agree we can close this now?

**Comment by filipefurtad0** [comment]
Yes, thanks @sigmundpetersen - it should be fixed by https://github.com/openfoodfoundation/openfoodnetwork/pull/10936 and https://github.com/openfoodfoundation/openfoodnetwork/pull/10933. Closing.
